app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1891,8 +1891,12 @@ public void onFullscreenStateChanged(final boolean fullscreen) {
 
         if (fullscreen) {
             hideSystemUiIfNeeded();
+            viewPager.setVisibility(View.GONE);
         } else {
             showSystemUi();
+            if (playerService.getView().getVisibility() != View.GONE) {
```

Comment:
shouldn't this be != View.VISIBLE here?

what does this change intent to do here anyway?

Reply:
I'm checking (when fullscreen is turned off) if the player view is shown, for a sanity check. If it is, I show the viewpager again.
The whole idea of hiding it is to disable keyboard focusing to it when pressed. Also you could press "down" key on the hardware keyboard (in fullscreen view!) and scroll down to comments - which definitely looked like a bug. This is also fixed here.

Reply:
I've removed this check after considering it. It does not seem to be really needed. @Redirion

Reply:
If I remember correctly you could use arrow down key to reach description in fullscreen on android TV. Now it's 100% impossible which is a bad thing if you ask me